tokio-console/src/main.rs

```diff
@@ -214,3 +223,24 @@ async fn watch_details_stream(
         }
     }
 }
+
+fn gen_completion(install: bool, shell: Shell) -> color_eyre::Result<()> {
```

Comment:
nit, take it or leave it: I think it might make sense to put this function in the `config` module?

Reply:
fixed: https://github.com/tokio-rs/console/pull/336/commits/99f659f44532082f200e9cce0efa7996aaa0fb51